web/test/pages/profile.test.tsx

```diff
@@ -468,7 +468,7 @@ describe("profile", () => {
       chooseTab("numbers");
       expect(getEmployerIdValue()).toEqual("12-3456789");
       expect(getEntityIdValue()).toEqual("1234567890");
-      expect(getTaxIdValue()).toEqual("123456790");
+      expect(getTaxIdValue()).toEqual("123-456-790");
```

Comment:
We're not storing the number with hyphens in the user data, right? This is just the html display element?

Reply:
correct